File: 프로그래머스/unrated/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.js
function solution(t, p) {
    let count = 0;
    for(let i = 0; i < t.length - p.length + 1; i++) {
        const subStr = +t.slice(i, i+p.length);
        if(subStr <= Number(p)) count++;
    }
    return count;
}

File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.py
def solution(t, p):
    p_n = int(p)
    
    count = 0
    for i in range(len(t) - len(p) + 1):
        sliced = t[i:i+len(p)]
        sliced_n = int(sliced)
        
        if sliced_n <= p_n:
            count += 1
            
    return count
            
    